recipe/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "pysam" %}
+{% set version = "0.22.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pysam-developers/pysam/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 61b3377c5f889ddc6f6979912c3bb960d7e08407dada9cb38f13955564ea036f
+  patches:
+    - patches/0001-Unvendor-deps.patch
+
+build:
+  number: 0
+  # https://github.com/samtools/htslib/issues/86
+  skip: True  # [win]
```

Comment:
The vendored version of `htslib` is [1.18](https://github.com/pysam-developers/pysam/blob/08de757a39ab42e867fa188faa860a5fd0b648c1/htslib/version.sh#L27). The Windows support was added in v1.6, see https://github.com/samtools/htslib/commit/75d9f0d9a973898686a51b60277f239c7e729ea5. 

You can try to build with:
```
build:
  - m2w64-toolchain        # [win]
  - m2-patch        # [win]
host:
run: 
  - m2w64-gcc-libs   # [win]
```

Reply:
It's for snowflake, we don't need to support Windows and from the docs it doesn't seem "that" easy.